crypto/src/keypair.rs

```diff
@@ -0,0 +1,18 @@
+use blsttc::{PublicKeyShare, SecretKeyShare};
+use hex;
+
+// KeyPair is a public key share and a secret key share that each node holds.
+pub struct KeyPair {
+    pub pk_share: PublicKeyShare,
+    pub sk_share: SecretKeyShare,
+}
+
+impl KeyPair {
+    pub fn hex_public_key(&self) -> String {
+        hex::encode(self.pk_share.to_bytes())
```

Comment:
I'd suggest to keep pk and sk as bytes. It's more memory efficient and you don't need to care about encoding.

Reply:
Good point! It was intended to format input and output type as `String` for convenience, but `bytes` makes more sense.